pkgs/development/interpreters/ruby/default.nix

```diff
@@ -50,7 +50,7 @@ let
       # - If you run:
       #     ruby -e "puts RbConfig::CONFIG['configure_args']"
       # - In:
-      #     $out/${passthru.libPath}/${stdenv.targetPlatform.system}/rbconfig.rb
+      #     $out/${passthru.libPath}/${stdenv.hostPlatform.system}/rbconfig.rb
```

Comment:
I'm not positive if this comment required updating.

Reply:
I think you made the right call and it should be.